src/article-editor.jsx

```diff
@@ -188,6 +193,11 @@ var ArticleEditor = React.createClass({
                         </div>
                     </div>
                 ];
+
+                if (section.content) {
+                    searchIndex -= Util.countOccurences(section.content, searchString);
```

Comment:
I find doing this logic inside a .map in jsx a bit hard to follow, if just for the reason that there's a lot of template code around it.

Maybe we could precompute this as a second array before the loop? such as

```javascript
var sectionSearchCount = 0;
var cumulativeSectionSearchCounts = _.map(sections, (section) => {
    sectionSearchCount += Util.countOccurences(section.content || "", searchString);
    return sectionSearchCount;
});
```

and then pass in `searchIndex - cumulativeSectionSearchCounts[i]` as the `searchIndex` prop